espresso/environment/optitmism_espresso_test_helpers.go

```diff
@@ -230,14 +230,13 @@ func (e EspressoDevNodeContainerInfo) Stop() error {
 // is meant to be.
 var ErrUnableToDetermineEspressoDevNodeSequencerHost = errors.New("unable to determine the host for the espresso-dev-node sequencer api")
 
-func (l *EspressoDevNodeLauncherDocker) StartDevNet(ctx context.Context, t *testing.T, L1finalizedDistance uint64, options ...DevNetLauncherOption) (*e2esys.System, EspressoDevNode, error) {
+func (l *EspressoDevNodeLauncherDocker) StartDevNet(ctx context.Context, t *testing.T, options ...DevNetLauncherOption) (*e2esys.System, EspressoDevNode, error) {
 	originalCtx := ctx
 
 	sysConfig := e2esys.DefaultSystemConfig(t, e2esys.WithAllocType(config.AllocTypeEspresso))
 
 	// Set a short L1 block time and finalized distance to make tests faster and reach finality sooner
 	sysConfig.DeployConfig.L1BlockTime = 2
```

Comment:
In all of the tests we've written thus far, this `L1BlockTime` is being explicitly set to `0`.  Do we want to change this to be `0` by default, so we don't actually need to pass the configuration to set it to `0` in all of our tests?

Reply:
We should confer with @philippecamacho to see what he thinks, and to double check that we should be setting the `L1BlockTime` for all of the tests to `0`.

Reply:
Could you point to where the `L1BlockTime` is set to `0` in our tests? I searched for `L1BlockTime`, and this line seems to be the only place under `espresso` where we set `L1BlockTime`.

Agree that if we set it to `0` everywhere else, it makes sense to default it to `0` instead of `2` here as well. But I just couldn't find where it's set to `0` yet.

Reply:
Oh, this is my bad.  I meant to refer to the `L1FinalizedDistance` instead of the `L1BlockTime`, but I must have made a mistake when creating the comment as the variable assignment was already removed.

Reply:
Got it! As discussed in the stand-up, I added a separate task to improve the `L1FinalizedDistance` setting.